components/ai_chat/resources/page/components/notices/notices.module.scss

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+.notice {
+  position: relative;
+  border-radius: var(--leo-radius-m);
+  background-color: var(--leo-color-container-highlight);
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  overflow: hidden;;
+  color: var(--leo-color-text-secondary);
+}
+
+.illustration {
+  flex: 0 0 auto;
+  width: 40%;
+  max-width: 260px;
+  background-color: #FF815C;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 3%;
+
+  img {
+    width: 100%;
+    height: auto;
+  }
+}
+
+.content {
+  flex-grow: 1;
+  padding: 16px;
+
+  h4 {
+    margin: 0 0 4px 0;
+    // width of close button, make sure no overlap
+    padding-right: 20px;
+    display: block;
+    overflow: hidden;
+    overflow-wrap: break-word;
+    font: var(--leo-font-heading-h4);
+    color: var(--leo-color-text-primary);
+  }
+
+  p {
+    margin: 0 0 4px 0;
+    font: var(--leo-font-default-regular);
+  }
+
+  a {
+    font: var(--leo-font-default-link);
+    color: inherit;
+  }
+}
+
+.closeButton {
+  position: absolute;
+  top: 16px;
+  right: 16px;
```

Comment:
optional: Use Nala spacing tokens in this file

Reply:
done